README.md

```diff
@@ -90,6 +90,16 @@ The samples resource maintains the following conditions in its status:
 See https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusteroperator.md#conditions for how these
 Cluster Operator status conditions are managed.
 
+# Disconnected mirroring assistance
+
+A `ConfigMap` named `imagestreamtag-to-image` is created in the Samples Operator's namespace that contains and 
+entry for each `ImageStreamTag`, where the value of the entry is the image used to populate that `ImageStreamTag`.
+
+The format of the key for each entry in the `data` field in the `ConfigMap` is `<imagestreamname>_<imagestreamtagname>`.
```

Comment:
so frustrating that colons aren't allowed as configmap keys :(

Reply:
+1